packages/common/src/operations.generated.ts

```diff
@@ -4,70 +4,6 @@ import { GraphQLClient } from 'graphql-request';
 import * as Dom from 'graphql-request/dist/types.dom';
 import gql from 'graphql-tag';
 import { graphql, ResponseResolver, GraphQLRequest, GraphQLContext } from 'msw'
-export type StocktakeQueryVariables = Types.Exact<{
```

Comment:
removed the stocktake operations from common, breaking things. 1 step back two steps forward..?